packages/web/.eslintrc.cjs

```diff
@@ -12,5 +12,13 @@ module.exports = {
   plugins: ['react-refresh', 'prettier', 'jest'],
   rules: {
     'prettier/prettier': 'error',
+    '@typescript-eslint/no-unused-vars': [
+      'warn', // or "error"
```

Comment:
This still means that we will fail on unused variables, just not when they start with the below patterns, correct?

Reply:
Correct, because max-warnings is set to zero it will still fail a build but will display as a warning instead of an error, but if you want it to also display as an error I can flip it over

Reply:
As long as it fails the build (and doesn't start with a `_`), I don't care if it's warning or error.

Reply:
yep build always fails if an unused var isn't prefixed with `_name` or just named `_`